util/util_test.go

```diff
@@ -182,3 +183,25 @@ func (s *TestSuite) TestGetValidMountPoint(c *C) {
 		}
 	}
 }
+
+func TestTimestampAfterTimestamp(t *testing.T) {
+	tests := map[string]struct {
+		before string
+		after  string
```

Comment:
Hmm.  Perhaps you agree with me, because you order the structure as before, after.  And so are the arguments to the function under test (tc.before, tc.after).  But then the actuallyAfter case has "before" after "after"... Oy.

Reply:
Hopefully I clarified this in the same commit I used for https://github.com/longhorn/longhorn-manager/pull/2432/files#r1462198405.